### ISSUE TITLE
Using the @v1 seems to fail

### ISSUE THREAD

**Issue by mightyiam** [msimerson/node-lts-versions#6]
### Expected behavior

I should be able to use as
```
        uses: msimerson/node-lts-versions@v1
```

### Observed behavior

Doing that results in some failure to find `node-version-data`.

### Steps to reproduce

Do this:

```
        uses: msimerson/node-lts-versions@v1
```

**Comment by msimerson** [comment]
Yep, version 1 was broken. Use `v1.2.2` instead. Maybe I should publish it as v2? 🤷‍♂️

**Comment by mightyiam** [comment]
Wait a moment. I don't expect `@v1` to refer to a specific version. I expect it to refer to the most recent version in which the major version is 1. That's how most GitHub Actions I use work as far as versions are concerned. Now, I don't know how this works, but perhaps it's as simple as not having a `v1` tag present in the repo.

**Comment by msimerson** [comment]
>  I expect it to refer to the most recent version in which the major version is 1

Oh.

**Comment by msimerson** [comment]
The way it works (see docs link) is that the developer takes an extra step to "move" the v1 tag as new v1.* semver release are made.

**Comment by mightyiam** [comment]
Thank you!
